app/Resources/views/article.html.twig

```diff
@@ -316,6 +316,35 @@
 
     {%- endif %}
 
+    <script>
+        {
+            "@context": "https://schema.org",
+            "@type": "Report",
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": "{{ url('article', {id: item.id}) }}"
+            },
+            "headline": "{{ item.fullTitle|striptags }}",
+            "image": [
+                "https://iiif.elifesciences.org/journal-cms/cover%2F2019-07%2F46846-a_striking_image.jpg/0,179,1800,543/1114,336/0/default.jpg"
+            ],
+            "datePublished": "{{ item.publishedDate|date('Y/m/d', 'UTC') }}",
```

Comment:
`UTC` shouldn't be needed, IIRC all dates are UTC anyway.

Reply:
As discussed this is being used further up the page.